app/facades/food_search_facade.rb

```diff
@@ -0,0 +1,8 @@
+class FoodSearchFacade
+  def self.get_foods(query)
+    FoodService.get_food_details(query)
+    json[:foods][0..9].map do |food_data|
```

Comment:
Consider using the API's query parameter of `pageSize` to limit the food response to 10 instead of having to use Ruby (just a bit cleaner)